packages/jest-resolver/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@web-stories-wp/jest-resolver",
+  "description": "Custom Jest resolver for monorepos",
+  "private": true,
```

Comment:
Should this be `"private": true` ?

Reply:
Yes this does not need to published; it can stay private